### ISSUE TITLE
Responsiveness for "Personal" map

### ISSUE THREAD

**Issue by GreenWithMV** [projectunic0rn/travel-map-ui#74]
Document Issue specifics: 

Mobile:
-select city timing has rounded corners, text is cut off.
-hamburger menu dropdown has small spacing for "personal, friends, profile"
-search/type city "x" is too low (not centered)
-"share" icon has a yellow box-shadow instead of green
-Tap Cities popup slightly cut-off

**Comment by hoangviet1993** [comment]
Can I take this item?
> -select city timing has rounded corners, text is cut off.

**Comment by GreenWithMV** [comment]
> Can I take this item?
> 
> > -select city timing has rounded corners, text is cut off.

Yes, sounds great. If you also want to take a stab at the hamburger menu one and the Tap Cities popup that would be awesome!

**Comment by hoangviet1993** [comment]
@GreenWithMV can you clarify 

>  has small spacing for "personal, friends, profile"

this part? Are you looking to make each of the item in the dropdown bigger, across all devices?

**Comment by GreenWithMV** [comment]
(We talked on Slack but just for history's sake). The spacing issue is seen on iOS Safari, perhaps not on other devices. I don't see it when I am in the Chrome Inspect mode even at an iPhone-sized screen. Basically, the line height for the first three links is smaller than it should be so they are bunched together.

**Comment by hoangviet1993** [comment]
@GreenWithMV Looks like it the hamburger menu is buggy across all pages on Safari. Would you like to create a separate issue thread for the hamburger menu?

**Comment by hoangviet1993** [comment]
Can we close this issue? @GreenWithMV
